spring-kafka-test/src/main/java/org/springframework/kafka/test/assertj/KafkaConditions.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright 2016 the original author or authors.
+ * Copyright 2017 the original author or authors.
```

Comment:
M-m-m. Sorry, if I was unclear.
We leave the first year as is and change (or add) only the current one.